### PR TITLE
Some UI enhancements and typo fixes

### PR DESCRIPTION
**Pull request by romanogit**
Suggested fixes
CardZoom.jsx: added a key to DIV loop to avoid browser warning in DEV mode
DeckSummary.jsx: add a divStyle at the bottom to allow better resizing of deck's card list
PlayerRow.jsx/gameboard.less: handling purged pile the same way as others to allow manual move of purge cards
Profile.jsx: fixed an issue which was not allowing email to be validated and causing page warning
MotherNorthelle.js: standardized aember wording to amber, since nowhere else aember is spelled this way
NotFinishedWithYou.js: fixed a typo

### CODE REVIEW

File: client/Components/Decks/DeckSummary.jsx [hunk 1]
```diff
@@ -157,6 +163,7 @@ class DeckSummary extends React.Component {
                         { cardsToRender }
                     </div>
                 </div>
+                <div style={ divStyle } />
```
Comment (cryogen) [inline client/Components/Decks/DeckSummary.jsx]: This should be in the .css/.less if it's not dynamic

Comment (romanogit) [reply client/Components/Decks/DeckSummary.jsx]: Ok, I will fix it.

**Comment by romanogit** [comment]
Changed the way I fixed #283 to also handle #342 :
image

**Comment by romanogit** [comment]
Closing this pull request as it has too many commits and it is holding trivial changes

**Pull request closed without merging**